### DISCUSSION TITLE
info

### DISCUSSION THREAD

**Discussion by francoborgia** [reworkd/AgentGPT#639] [Q&A]
Hi, I installed everything via docker.
when i connect to my localhost and try to use a gpt agent query it gives me the following  error  There was an issue getting a response from the AI model.
i hope you can help me

**Comment by tamaosaka** [comment]
Hello,

Found the file.
Feel free to let me know if anything is still missing

Sorry if it's unrelated to the thread.
Thanks and bestregards.


-----

screenshots is create inntrnet song pic.
When a task comes out, I think about it, and when I do the task, it's another task. was.
I was worried about being alone.Sometimes I wanted people to ask me, "What do you think of this idea?"

Agent was thinking a lot about orchestras, rock bands, folk, and so on.
But I remember saying that there is height and brightness.
How about the weight (gravity) and the elements of oxygen, carbon dioxide, nitrogen, and hydrogen?


-----

![Screenshot_20230501-045342.png](https://github.com/reworkd/AgentGPT/assets/131488149/a53d560d-0467-4074-aa4d-f10a22bd9727)

melody scanner is AgentGPT idea.
I was very surprised.
Sound does not exist on the Internet.


![Screenshot_20230501-021317.png](https://github.com/reworkd/AgentGPT/assets/131488149/b6afb23c-2374-46c7-85f9-13174c3408ed)

I will add it later.

-----

![Screenshot_20230502-113048~2.png](https://github.com/reworkd/AgentGPT/assets/131488149/1f2cc6b5-139e-4305-9ea2-6b05a4eb96de)

![Screenshot_20230502-124215.png](https://github.com/reworkd/AgentGPT/assets/131488149/f8377ebb-a4a3-4c02-b66f-96cd5a494e59)

titte,
power saving

goal,
is cipher, pattern and image processing an application that reduces the file size using such as make.

dont zip tech nothing filesize change mini.
refer below please,
before ------
- for example, 
if this analogy is also short(binary).

after -----
- for if this is also short.

or erase the alphabet (aiueo)
- fr xmpl, 
  f ths nlgy s ls shrt

Do it before sending or saving, and AgentGPT when opening or sending

english is use byte a little.
but kanji use english*2.


- i think cookie, java and steruss for browser and more apps option(authority).

- i thnink need create want app not program.
but i dont programming.
no code too.
wait 30 years.
want desktop pc now.

![Screenshot_20230602-143446~2.png](https://github.com/reworkd/AgentGPT/assets/131488149/8de7f8ac-ddc0-434c-a1d7-090107c37639)

pgp.

that's all.I'll fill in the gaps later.
Thanks anf bestregards.
